src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs

```diff
@@ -89,31 +87,6 @@ public async Task ShowAsync_VerifyNotificationIsCreatedWithExpectedParameters(bo
             notification.Actions.Last().CommandText.Should().Be(MigrationStrings.MigrationPrompt_LearnMoreButton);
         }
 
-        [TestMethod]
-        public async Task ShowAsync_ShowNotificationIsCalledOnMainThread()
```

Comment:
The migration prompt class no longer needs to do any explicit thread handling, so this thread is redundant.